### PR TITLE
add grid and queries

### PR DESCRIPTION
**Pull request by Pozdnya**
- [DEMO LINK](https://pozdnya.github.io/layout_catalog_grid/)

### CODE REVIEW

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -1,22 +1,32 @@
...
+@mixin query ($cols) {
+    grid-template-columns: repeat($cols, 200px);
+}
 
-  max-width: 944px;
-  margin: 0 auto;
+.catalog {
+  display: grid;
+  justify-content: center;
+  align-content: center;
```
Comment (AlexRog01) [inline src/styles/_catalog.scss:8]: No alignment needed here.
image

**Review by AlexRog01 — APPROVED** [review]
Personally I changed the width of the catalog and didn`t align the content inside.

File: src/styles/_catalog.scss (hunk 1 above)
Comment (Pozdnya) [reply src/styles/_catalog.scss:8]: add  `align-content: start;` for `.catalog`

File: src/styles/_catalog.scss [hunk 2]
```diff
@@ -1,22 +1,32 @@
...
+  grid-template-columns: repeat($cols, 200px);
+}
 
-  max-width: 944px;
-  margin: 0 auto;
+.catalog {
+  display: grid;
+  justify-content: center;
+  align-content: start;
+  gap: 48px;
```
Comment (liliia-livandovskaya) [inline src/styles/_catalog.scss:9]: you need to set the default `grid-template-columns` to the catalog

File: src/styles/_catalog.scss [hunk 3]
```diff
@@ -1,22 +1,32 @@
...
+  @media (max-width: 1024px) and (min-width: 768px) {
+    @include query(3);
+  }
+
+  @media (max-width: 768px) and (min-width: 488px) {
+    @include query(2);
+  }
+
+  @media (max-width: 488px) {
+    @include query(1);
```
Comment (liliia-livandovskaya) [inline src/styles/_catalog.scss:30]:
```suggestion
  @media (min-width: 1024px) {
    @include query(4);
  }

  @media (min-width: 768px) {
    @include query(3);
  }

  @media (min-width: 488px) {
    @include query(2);
```
and add `grid-template-columns: 1, 200px` to the `catalog` as a default value

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
it looks weird

Screenshot 2023-03-06 at 10 46 39

pay attention to the example of how your catalog should look and apply needed changes 

Screenshot 2023-03-06 at 10 47 01

**Pull request closed without merging**